### PR TITLE
Use nix 0.18 and winapi-wsapoll 0.1.1

### PR DESCRIPTION
**Pull request by eduardosm**

### CODE REVIEW

**Comment by eduardosm** [comment]
What's going on with appveyor? pr check passes but branch check fails

https://ci.appveyor.com/project/psychon/x11rb/builds/35533815#L648

**Comment by eduardosm** [comment]
Now branch has passed and pr failed.

**Comment by psychon** [comment]
For future reference, here is the appveyor error:
```
cargo check --verbose --all-targets --features all-extensions,cursor,image
    Updating crates.io index
error: failed to get `roxmltree` as a dependency of package `x11rb-generator v0.0.0 (C:\projects\x11rb\generator)`
Caused by:
  failed to load source for dependency `roxmltree`
Caused by:
  Unable to update registry `https://github.com/rust-lang/crates.io-index`
Caused by:
  failed to fetch `https://github.com/rust-lang/crates.io-index`
Caused by:
  error inflating zlib stream; class=Zlib (5)
Command exited with code 101
```
This sounds like a transient issue. I'll restart the AppVeyor build and hope that it succeeds.

Travis basically succeeded (only the expected clippy error that I've been ignoring for a while and which supposedly disappears with the next stable Rust release).

**Comment by pickfire** [comment]
@psychon Can we create a new release since this is updated? I want to de-duplicate the dependencies.

**Comment by psychon** [comment]
Uhm... I now no reason why not. @eduardosm do you know one?

However, I might not have much time this month to actually do a release. Sorry. Feel free to open an issue about that and pick me around the 20th.

**Comment by pickfire** [comment]
Maybe I can send a pull request for a release?

**Comment by psychon** [comment]
I cannot stop you from sending PRs. ;-)

The process for doing a release is documented in https://github.com/psychon/x11rb/blob/master/doc/making_a_release.md. "Update the changelog" is not really self-explanatory, but I read through `git log LAST_RELEASE..` and just left out uninteresting stuff, e.g. a bug fix for a bug that was not yet part of the previous release.

Thanks!

**Comment by eduardosm** [comment]
> Uhm... I now no reason why not. @eduardosm do you know one?

Nope, I don't see any reason not to release a new version.

**Comment by psychon** [comment]
@pickfire Dunno if you already started on this, but thanks to corona I'll give this a try. Sorry if this wastes some of your work.

**Comment by pickfire** [comment]
Ah, I haven't started this. Wait, you got corona? Sad to hear that but hope you gets well soon.

**Comment by psychon** [comment]
Thanks. :-)

No, I did not. Uhm... I'm not sure how it's handled elsewhere, but here it is something like: If there are more than 50 cases per 1.000 people in a county(?), that county is a risk area. When checking in to a hotel / vacation home and you are from a risk area, you need a negative covid19 test that is not older than two days. Covid19 tests are expensive and hard to come by, because everyone with a good reason to get one is before you in line for the test. And in a risk area, there are many people with good reason to get a test.

TL;DR: Covid19 just made it harder to travel (in country) which only means cancelled plans. No actual infection here.
